packages/devtools-components/src/tests/tree.js

```diff
@@ -653,14 +778,27 @@ function getTreeNodes(wrapper) {
   return wrapper.find(".tree-node");
 }
 
-function simulateKeyDown(wrapper, key) {
+function simulateKeyDown(wrapper, key, options) {
   wrapper.simulate("keydown", {
     key,
     preventDefault: () => {},
-    stopPropagation: () => {}
+    stopPropagation: () => {},
+    ...options
   });
 }
 
+function renderItemWithFocusableContent(x, depth, focused, arrow) {
+  let children = [{}, arrow, focused ? "[" : null, x, focused ? "]" : null];
```

Comment:
nit: we should put the closing bracket after the `if (x === "L")` block so it "wraps" the whole line.

Reply:
also, I'd exclude the first object from the array since it's the `dom.div` option object, and not a children